addons/arsenal/functions/fnc_buttonLoadoutsSave.sqf

```diff
@@ -155,7 +155,7 @@ switch (GVAR(currentLoadoutsTab)) do {
 
         if (is3DEN) then {
 
-            private _sameNameLoadoutsList = _data select {_x select 0 == _editBoxContent};
+            private _loadoutIndex = (+(_data)) findId {(_x select 0) == _editBoxContent};
```

Comment:
`findId` -> `findIf` ?

Reply:
RIP

Reply:
Why `(+(_data))` ? Why copy the array. Don't see any need.

Reply:
@alganthe had me deep copy everything

Reply:
*slaps @alganthe*

Reply:
hey no, I didn't told you to deep copy EVERYTHING, just the namespace var that is used to store loadouts (depends on which tab is active).